clients/vault/src/metrics.rs

```diff
@@ -265,23 +269,25 @@ impl PerCurrencyMetrics {
 	// }
 
 	pub async fn initialize_values(parachain_rpc: SpacewalkParachain, vault: &VaultData) {
-		// let bitcoin_transactions = match vault.btc_rpc.list_transactions(None) {
-		//     Ok(x) => x
-		//         .into_iter()
-		//         .filter(|x| x.detail.category == GetTransactionResultDetailCategory::Send)
-		//         .collect(),
-		//     Err(_) => vec![],
-		// };
+		let bitcoin_transactions =
```

Comment:
Should be renamed to `stellar_transactions`

Reply:
Done